umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -37,12 +68,7 @@
    (if (and (nil? description-index)(nil? url-content-type-index)
             (nil? type-index) (nil? subtype-index))
     {:Description description} ; Description not formatted like above, so just description
-    {:Description (when description-index
-                   (let [desc (subs description
-                               description-index
-                               (or url-content-type-index type-index subtype-index
-                                   checksum-index (count description)))]
-                    (str/trim (subs desc (inc (.indexOf desc ":"))))))
+    {:Description (convert-description-string-to-map description description-index)
```

Comment:
```suggestion
    {:Description (get (convert-description-string-to-map description description-index) "Description")
     :URLContentType (get (convert-description-string-to-map description url-content-type-index) "URLContentType")
     :Type (get (convert-description-string-to-map description type-index) "Type")
     :Subtype (get (convert-description-string-to-map description subtype-index) "Subtype")
     :Checksum (get (convert-description-string-to-map description checksum-index) "Checksum")})))
```

Reply:
The :Description value should be just the description and not the map

The rest of the function also needs to be refactored with your new function.